client/state/happychat/middleware.js

```diff
@@ -30,6 +29,38 @@ const eventMessage = {
 	HAPPYCHAT_FOCUS: 'Started looking at Happychat',
 };
 
+// This function creates a stub `Connection` object stub that dynamically loads the chunk
```

Comment:
Should this read `This function creates a stub `Connection` object that dynamically loads the chunk` instead?

Reply:
Yes, fixed 👍